app/components/supplementary-file-browser/template.hbs

```diff
@@ -0,0 +1,65 @@
+{{#file-renderer download=selectedFile.links.download
+    width="99%" height="700"}}
+{{/file-renderer}}
+
+<div class="row">
+    <div class="col-sm-6">
+        <p class="f-w-lg p-b-md">{{selectedFile.name}}</p>
+    </div>
+    <div class="col-sm-6 text-nowrap supplemental-downloads">
+        {{#if (eq selectedFile.id preprint.primaryFile.id)}}
+            <a class="btn btn-default btn-sm" href={{fileDownloadURL}}> {{t "content.share.download_preprint"}}</a>
+        {{else}}
+            <a class="btn btn-default btn-sm" href={{fileDownloadURL}}> {{t "content.share.download_file"}} </a>
+        {{/if}}
+        <span class="p-sm"> {{t "content.version"}}: {{selectedFile.currentVersion}}</span>
+    </div>
+</div>
+
+{{#if hasAdditionalFiles}}
+    <section class="osf-box p-md m-t-sm m-b-lg">
+        {{#liquid-bind (slice-array files startIndex endIndex) use=scrollAnim as |list|}}
```

Comment:
I don't suppose there's a way that we could avoid using liquid fire? It's a very heavy library that honestly does not offer that much. [Animate.css](https://daneden.github.io/animate.css/) could be a potential alternative?